OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/install/oud.response

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Response files do not require copyright or license headers.

Reply:
Removed.